js/client-teambuilder.js

```diff
@@ -2100,12 +2104,17 @@
 					curSet.moves.push(line);
 				}
 			}
+			if (jsonTeams){
+			    for (var i=0;i<jsonTeams.length;i++){
```

Comment:
Spaces after semicolons in `for`s, and space before the opening brace.
(And spaces before and after binary operators, like mentioned above)